soccer/gameplay/plays/restarts/free_kickoff.py

```diff
@@ -0,0 +1,49 @@
+import play
+import behavior
+import robocup
+import skills.line_kick
+import tactics.defense
+import main
+import constants
+import enum
+import role_assignment
+
+
+class FreeKickoff(play.Play):
```

Comment:
This is technically also just a pivot kick although it is triggered on kickoff. Our current kickoff uses linekick so I would say if linekick is fixed we should remove this and use the normal kickoff.